pythoncz_tests/models/github_test.py

```diff
@@ -0,0 +1,432 @@
+import re
+import warnings
+
+import pytest
+import requests
+import responses
+from werkzeug.contrib.cache import NullCache
+
+from pythoncz.models import github as github_module
+from pythoncz_tests.models import github_test_fixtures as fixtures
+
+
+@pytest.fixture()
+def github():
+    """Provides the 'github' module with caching disabled"""
+    original_cache = github_module.cache
+    github_module.cache = NullCache()
+    yield github_module
+    github_module.cache = original_cache
+
+
+@pytest.fixture()
+def requests_mock():
+    """Provides mechanism for faking HTTP responses"""
+    with responses.RequestsMock() as mock:
+        yield mock
+    mock.reset()
```

Comment:
Why do you call `mock.reset()` here? It doesn't make sense to having to call some reset after with block - and indeed it is already included in `RequestsMock.__exit__()`: https://github.com/getsentry/responses/blob/master/responses.py#L381